runtime/drivers/registry.go

```diff
@@ -33,4 +33,6 @@ type Instance struct {
 	CreatedOn time.Time `db:"created_on"`
 	// UpdatedOn is when the instance was last updated in the registry
 	UpdatedOn time.Time `db:"updated_on"`
+	// EnviornmentVariables for the instance
+	Env *EnviornmentVariables `db:"env"`
```

Comment:
We should distinguish between user-provided overrides and project defaults. This will be useful both when reconciling updates to the `rill.yaml` file and when displaying env in UI.

Also, this should just be a `map[string]string`. The note in `EnvironmentVariables` docstring about supporting secrets, globals, etc. seems more to relate to the template injection than the underlying data model.

About secrets, we should assume any user-provided env var is a secret. We may store the map in the DB for cloud MVP, but later we should store the full map in Vault.



Suggested refactor:

```
// Env contains user-provided environment variables
Env map[string]string
// ProjectEnv contains default environment variables from rill.yaml
// (NOTE: This can always be reproduced from rill.yaml, so it's really just a handy cache of the values.)
ProjectEnv map[string]string
```